jest.config.base.js

```diff
@@ -3,8 +3,9 @@ module.exports = {
   verbose: true,
   testRegex: '(/src(/?[^/]*){0,5}/__tests__/).*Spec\\.jsx?$',
   modulePaths: ['<rootDir>/'],
+  transformIgnorePatterns: ['node_modules/(?!(@ima/core)/)'],
```

Comment:
Out of curiosity, why is this needed?

Reply:
The @ima/core is now pure esm package, so it needs to be transformed to cjs with other source files otherwise it fails on imports from @ima/core in Spec files.